website/docs/index.html.markdown

```diff
@@ -3,20 +3,63 @@ layout: "auth0"
 page_title: "Provider: Auth0"
 sidebar_current: "docs-auth0-index"
 description: |-
-  The Auth0 provider .
+  The Auth0 provider is an unofficial provider that leverages the Auth0
```

Comment:
Could we avoid the use of unofficial? Since we're trying to get this merged with official terraform providers we would have to update these references.

Reply:
Good point. I wet ahead and changed that.